packages/frontend/src/hooks/useMessageSentNotifier.ts

```diff
@@ -0,0 +1,63 @@
+import { useState, useCallback, useEffect } from 'react'
+import type { T } from '@deltachat/jsonrpc-client'
+
+// Simple global state for message notifications
+const messageListeners: Set<
+  (accountId: number, chatId: number, message: Partial<T.MessageData>) => void
+> = new Set()
+
+export function notifyMessageSent(
+  accountId: number,
+  chatId: number,
+  message: Partial<T.MessageData>
+) {
+  messageListeners.forEach(callback => callback(accountId, chatId, message))
```

Comment:
This should not throw if any of the callbacks throw.

Reply:
Oh, and if one listener throws, others need to be invoked still. So I guess `try { callback(...) } catch(e) {}` would work.

Reply:
Added a try - catch